11_structures/main.go

```diff
@@ -1,7 +1,11 @@
 package main
 
-/*
+import (
+	"fmt"
+	"math"
```

Comment:
I did import math to be able to calculate the square root for the triangle area.
Had attempted to get it manually, but was a bit time consuming and I think it will also just be an approximate value.

Reply:
That's valid :) I forgot that you would need it, good job on your part finding how to do it.